### DISCUSSION TITLE
neovim command bar gets hidden when using wezterm

### DISCUSSION THREAD

**Discussion by lovelindhoni** [wezterm/wezterm#5984] [Q&A] [answered]
### What Operating System(s) are you running on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

Mutter 46.4

### WezTerm version

20240811_002228_100cac77

### Ask your question!

When opening a file in neovim, the command bar at the bottom gets hidden at fullscreen. What is the config to fix this issue.
And when shrinking the window, there is some grey padding comes up. 
Running on fedora 40
Thanks ✨

![Screenshot from 2024-08-17 18-27-49](https://github.com/user-attachments/assets/25ae3853-0af5-4ed0-b7e6-c46b02b117b6)
![Screenshot from 2024-08-17 18-38-10](https://github.com/user-attachments/assets/12cd4cd3-f340-4891-9b99-0b18c3ed8d13)

**Comment by m2x07** [comment]
can you post your full config please?

**Comment by lovelindhoni** [comment]
Sure, here 
```lua
local wezterm = require("wezterm")

local config = wezterm.config_builder()

config.font_size = 16.0

config.window_decorations = "RESIZE"
config.enable_tab_bar = true

config.window_padding = {
	bottom = 20,
}

config.window_background_opacity = 0.8 -- Default opacity when not fullscreen

return config
```

**Reply by m2x07** [reply]
weird. i am not able to reproduce the issue. i'm on Gnome but using Xorg. maybe has to do something with wayland and/or mutter

**Comment by lovelindhoni** [comment, marked as answer]
It is wayland, I had to switch to Xorg to find this one. But using wezterm as an appimage looks fine on wayland
